docs/sources/whatsnew/whats-new-in-v11-3.md

```diff
@@ -0,0 +1,307 @@
+
+[Documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#add-field-from-calculation)
+
+## Explore Logs
+
+### The Explore Logs plugin is installed by default
+
+<!-- #proj-explorelogs-dev -->
+
+_Generally available in all editions of Grafana_
+
+With **Explore Logs** you can visualize and explore your logs to troubleshoot without having to write queries. It is easy to find spikes in your log volume, filter your logs and pinpoint problematic log lines.
+
+While **Explore Logs** is [GA in cloud](https://grafana.com/blog/2024/09/24/queryless-metrics-logs-traces-profiles/#explore-logs) and installed there by default already, with Grafana v11.3.0 it will be automatically installed on your Open Source or Enterprise instance as well. This will let you use Explore Logs alongside **Explore Metrics** without needing to install it manually.
+
+This is configured by the [`preinstall` configuration parameter](https://github.com/grafana/grafana/blob/9ece88d5852dceb90f83271e66902eece24f908f/conf/defaults.ini#L1748) in your Grafana configuration. For more information about Explore logs, refer to [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/simplified-exploration/logs/).
+
+## Alerting
+
+### Simplified query section for alert rule creation
+
+<!-- Sonia Aguilar -->
+
+_Generally available in all editions of Grafana_
+
+Use a simplified version of the query and alert conditions step to create Grafana-managed alert rules. The default options streamline rule creation with a cleaner header and a single query and condition. For more complex rules, switch to advanced options to add multiple queries and expressions.
+
+This feature is rolling out to Grafana Cloud over the next couple of weeks.
+
+Grafana Enterprise and OSS:
+
+To use this feature, enable the `alertingQueryAndExpressionsStepMode` feature toggle.
+
+{{< figure src="/media/docs/alerting/screenshot-simple-query-form-v11.3.png" max-width="800px" alt="Image shows the alert creation form in simple mode" >}}
+
+[Documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/alerting/alerting-rules/create-grafana-managed-rule/)
+
+## Data sources
+
+### GitHub App authentication for the GitHub data source
+
+<!-- #grafana-oss-big-tent -->
+
+_Generally available in all editions of Grafana_
+
+Authenticate the GitHub data source using a GitHub App, an alternative to personal access tokens (PATs). GitHub App authentication offers enhanced security by granting more granular permissions, reducing the risk of over-permissioning.
+
+For more information, refer to the [GitHub data source documentation](https://grafana.com/docs/plugins/grafana-github-datasource/latest/setup/token/#using-github-app-authentication) for detailed instructions on setting up GitHub App authentication.
+
+[Documentation](https://grafana.com/docs/plugins/grafana-github-datasource/latest/)
+
+## OnCall
```

Comment:
@devinrcheevers should we include OnCall improvements in the Grafana 11.3 release? These might not be applicable to grafana/grafana. CC @imatwawana

Reply:
My instant reaction is no, because OnCall isn't 'bundled' with Grafana OSS, but maybe I'm missing something.